docs/website/content/docs/v0.3/Guides/first-cluster.md

```diff
@@ -128,9 +127,9 @@ Note that there are several variables that should be set in order for the templa
 - `CONTROL_PLANE_SERVERCLASS`: The server class to use for control plane nodes.
 - `WORKER_SERVERCLASS`: The server class to use for worker nodes.
 - `KUBERNETES_VERSION`: The version of Kubernetes to deploy (e.g. `v1.19.4`).
-- `TALOS_VERSION`: This should correspond to the minor version of Talos that you will be deploying (e.g. `v0.9`).
+- `TALOS_VERSION`: This should correspond to the minor version of Talos that you will be deploying (e.g. `v0.10`).
   This value is used in determining the fields present in the machine configuration that gets generated for Talos nodes.
-  Note that the default is currently `v0.8`.
```

Comment:
default is still `0.8` atm to maintain backwards compatibility (it's in the CABPT)